daemons/dss-gs-copy-sfn/app.py

```diff
@@ -0,0 +1,17 @@
+import os
+import sys
+
+import domovoi
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), 'domovoilib'))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+import dss.stepfunctions.gscopyclient as gscopyclient
+import dss.stepfunctions.generator as generator
+
+app = domovoi.Domovoi()
+
+
+annotation_processor = generator.StateMachineAnnotationProcessor()
```

Comment:
Take a look at how @xbrianh's Vistitation SFN definition gets by without this. It cuts out a whole lot of machinery with very little additional code. It also is more flexible in the kinds of parameterizations it allows and is self-evident since it only uses standard dictionary and list literals.

Reply:
Actually, we don't need to do the annotation processor for this since it's single threaded.  copy-pasta. :(